contrib/diagnostic.py

```diff
@@ -19,7 +19,7 @@
 cv6.voltage(10)  # max
 
 while True:
-    oled.clear()
+    oled.fill(0)
```

Comment:
`fill()` is preferred over `clear()`?

Reply:
I updated the `clear()` method to include a `show()` call so we can clear the oled display with a single line of code. If you just want to clear the display buffer, use `fill(0)`.

Reply:
That's what I get for commenting before reading the whole thing.